### PR TITLE
Monorepo: New Arrow Glacier Releases

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: packages/client/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,43 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.2.0 - 2021-11-09
+
+### Experimental Merge Support
+
+This client release comes with experimental Merge support as specified in `EIP-3675` commit [504954e3](https://github.com/ethereum/EIPs/blob/504954e3bba2b58712d84865966ebc17bd4875f5/EIPS/eip-3675.md) and the Engine API [v1.0.0-alpha.2](https://github.com/ethereum/execution-apis/blob/v1.0.0-alpha.2/src/engine/interop/specification.md). This is the spec snapshot used for the [Merge Interop event](https://hackmd.io/@n0ble/merge-interop-spec) in Greece which happened in October 2021.
+
+For the curageous there are instructions [here](https://hackmd.io/B1nMKhIiRBWRtgP4-d5KHw) on how to connect the EthereumJS client with a Lodestar Eth 2.0 client and produce blocks together. Note that specifications are changing quickly though and a new testnet is already planned, so this is not guaranteed to work.
```
Comment (ryanio) [inline packages/client/CHANGELOG.md:15]:
```suggestion
For the courageous there are instructions [here](https://hackmd.io/B1nMKhIiRBWRtgP4-d5KHw) on how to connect the EthereumJS client with a Lodestar Eth 2.0 client and produce blocks together. Note that specifications are changing quickly though and a new testnet is already planned, so this is not guaranteed to work.
```

File: packages/client/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,43 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+- PR [#1509](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1509) (first Engine API steps, JSON RPC block parameter alignments, HF-by-TD fixes)
+- PR [#1512](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1512) (continued Engine API, `--rpcEngine` CLI option, Taunas testnet, temporary eth_getLogs RPC mockup)
+- PR [#1530](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1530) (dedicated Engine API port, Pithos testnet, Docker)
+
+### An Update on Mainnet
+
+The 1,230,833 block consensus bug we had in the `v0.1.0` release has been fixed in the VM dependency (see PR [#1516](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1516) and PR [#1524](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1524)), so sync on `mainnet` will now continue from this block onwards.
+
+The adventure "Mainnet sync" now comes to a halt though around the Shanghai dDoS attacks mainnet blocks ranging from 2,286,910 to 2,717,576 where we realize that our VM is just not optimized enough to hold through these various edge case scenarios.
```
Comment (ryanio) [inline packages/client/CHANGELOG.md:27]: adventure?

File: packages/tx/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,49 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+Before:
+
+```shell
+Invalid Signature: s-values greater than secp256k1n/2 are considered invalid
+```
+
+New:
+
+```
+Invalid Signature: s-values greater than secp256k1n/2 are considered invalid (tx type=1 hash=0xbf78bd19410294cb3c08fbbbdd675b4bd79e46b96b38e850091f5c03e0571be0 nonce=0 value=0 signed=true hf=london gasPrice=0 accessList=0 (size))
```
Comment (ryanio) [inline packages/tx/CHANGELOG.md:40]: just a note, I found the `(size)` confusing after `accessList`, maybe later we can update it to `accessListCount=0`?

**Review by ryanio — CHANGES_REQUESTED** [review]
LGTM! just a few comments

File: packages/client/CHANGELOG.md (hunk 2 above)
Comment (ryanio) [reply packages/client/CHANGELOG.md:27]: did you mean
> The "Mainnet sync" adventure now comes...

Comment (ryanio) [reply packages/client/CHANGELOG.md:27]:
```suggestion
The "Mainnet sync" adventure now comes to a halt though around the Shanghai dDoS attacks mainnet blocks ranging from 2,286,910 to 2,717,576 where we realize that our VM is just not optimized enough to hold through these various edge case scenarios.
```

**Review by ryanio — DISMISSED** [review]
lgtm!

File: packages/tx/CHANGELOG.md (hunk 3 above)
Comment (ryanio) [reply packages/tx/CHANGELOG.md]: have updated in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1555/commits/9304010d672a517f13ecb9048dab9176e6bfa09a

**Review by ryanio — APPROVED** [review]

**Comment by holgerd77** [comment]
Just published the following releases on npm:

- `@ethereumjs/common` [v2.6.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fcommon%402.6.0)
- `@ethereumjs/tx` [v3.4.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%403.4.0)
- `@ethereumjs/block` [v3.6.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblock%403.6.0)
- `@ethereumjs/blockchain` [v5.5.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblockchain%405.5.0)
- `@ethereumjs/vm` [v5.6.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fvm%405.6.0)
- `@ethereumjs/client` [v0.2.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fclient%400.2.0)

**Pull request merged**